tests/support/pytest/transport.py

```diff
@@ -275,7 +306,12 @@ def __enter__(self):
 
     def __exit__(self, *args):
         # Publish a payload to tell the collection it's done processing
-        self.publish({"tgt_type": "glob", "tgt": "*", "jid": -1, "stop": True})
+        attempts = 300
```

Comment:
Should this throw an error if we go through all attempts?

Reply:
I fixed the missing error handling.

Reply:
@fanaticize Thanks. Unfortunately there's a linter error now.